docOps/docs/libraries/nodejs/examples.md

```diff
@@ -11,7 +11,20 @@ In `nodejs` binding, the `Client` instance is instantiated and optionally config
 The library is designed to automatically choose a starting IOTA node based on the network type one would like to participate in: `testnet` or `mainnet`. So very simplistic example how to connect to [IOTA testnet](https://chrysalis.docs.iota.org/testnet.html) is the following one:
 
 ```javascript
-{{#include ../../../bindings/nodejs/examples/01_get_info.js}}
```

Comment:
I know this #include doesn't work with docusauraus, but is there not something similar that works? Looks like https://github.com/dotansimha/remark-import-partial could work, maybe worth a try, otherwise it's probably easy to forget to update one